src/Main/Main.js

```diff
@@ -0,0 +1,9 @@
+import JobsContainer from "../JobsContainer/JobsContainer";
+
+const Main = (props) => {
+    return (
+        <JobsContainer apiFetch={props.apiFetch}/>
+    )
+}
+
+export default Main
```

Comment:
doesn't seem like there's any reason for this component to exist? with or without it the resulting HTML, CSS & JS generated for the website would be the same